### PR TITLE
Configure eleventy dev server to serve content over HTTPS

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-blog/issues/573

### CODE REVIEW

File: eleventy.config.js [hunk 1]
```diff
@@ -96,33 +96,17 @@ module.exports = function configure(eleventyConfig) {
...
-          bs.addMiddleware('*', (req, res) => {
-            const content_404 = fs.readFileSync('./build/error.html');
-            // Provides the 404 content without redirect.
-            res.write(content_404);
-            // Add 404 http status code in request header.
-            res.writeHead(404);
-            res.end();
-          });
-        },
-      },
```
Comment (willdurand) [inline eleventy.config.js:111]: see https://github.com/mozilla/addons-blog/issues/575

File: eleventy.config.js [hunk 2]
```diff
@@ -96,33 +96,17 @@ module.exports = function configure(eleventyConfig) {
...
-    };
-
     if (process.env.USE_HTTPS === '1') {
-      browserSyncConfig = {
-        ...browserSyncConfig,
+      const serverOptions = {
         host: 'example.com',
```
Comment (diox) [inline eleventy.config.js:101]: Do you remember if there was a reason `example.com` was used and not `localhost` or something using the reserved `.test` tld ? That makes the setup annoying, need to remember to add `127.0.0.1 example.com` to `/etc/hosts` ...

**Review by diox — APPROVED** [review]
The `README.md` should also be updated to remove reference to `browsersync`

File: eleventy.config.js (hunk 2 above)
Comment (willdurand) [reply eleventy.config.js:101]: yeah, `example.com` is hardcoded in gecko: https://searchfox.org/mozilla-central/rev/95ec620fda8ad6e4096cac1f1a9db8653669b31d/toolkit/mozapps/extensions/AddonManager.sys.mjs#59,61

**Pull request merged**